.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
     python: 3.7
   - env: TOXENV=py37-dj2-wt23
     python: 3.7
+  - env: TOXENV=py37-dj21-wt23
+    python: 3.7
```

Comment:
Hey @sekanitembo. We dropped 2.1 support on purpose because it's no longer supported by Django, so this addition should be reverted. I should actually remove the Django 2.0 tests too, as that is also no longer receiving security updates either (I will take care of that in a separate PR)